Code/GraphMol/MolDraw2D/Wrap/testMolDraw2D.py

```diff
@@ -12,66 +12,66 @@
 
 class TestCase(unittest.TestCase):
```

Comment:
General comment: whatever tool you are using to format Python code doesn't follow the indentation standards that the rest of the RDKit uses. Please either change those settings or make sure it doesn't reformat code.
I've fixed this in my PR

Reply:
Sorry.  I use CLion for C++ development and it isn't very good with Python, and on closing files it reformats everything with clang.

Reply:
ick... :-)

Reply:
I should remember, emacs is the only true path :-)